assets/src/components/vehiclePropertiesCard.tsx

```diff
@@ -7,16 +7,18 @@ import React, {
 import { joinTruthy, className as classNames } from "../helpers/dom"
 import { useCurrentTimeSeconds } from "../hooks/useCurrentTime"
 import { useNearestIntersection } from "../hooks/useNearestIntersection"
-import { Vehicle } from "../realtime"
+import { isGhost, isVehicle } from "../models/vehicle"
+import { Vehicle, VehicleOrGhost } from "../realtime"
 import { CloseButton } from "./closeButton"
-import StreetViewButton, {
-  GeographicCoordinateBearing,
-} from "./streetViewButton"
+import StreetViewButton from "./streetViewButton"
 import { VehicleRouteSummary } from "./vehicleRouteSummary"
 
 interface VehicleProp {
   vehicle: Vehicle
 }
+interface VehicleOrGhostProp {
+  vehicle: VehicleOrGhost
```

Comment:
thought (non-blocking): I like how `mapPage` now uses the name `selectedVehicleOrGhost` wherever type is `VehicleOrGhost`. I wonder if it is worth following that same naming convention in more places?

Reply:
Thanks! PTALA fc35c035 🙏🏻